### DISCUSSION TITLE
Logging with logstash and logback in a docker container

### DISCUSSION THREAD

**Discussion by le8266** [logfellow/logstash-logback-encoder#487] [Q&A] [answered]
I will use the LogstashTcpSocketAppender in a logback configuration to send log messages to logstash. The java application works, but if I create a docker image and start the container then the messages don't arrive logstash. What can be the reason for this behavior? I wrote a sample application. You can find the full question [here](https://stackoverflow.com/questions/61998203/logging-with-logstash-and-logback-in-a-docker-container).

**Comment by philsttr** [comment, marked as answer]
Addressed on stackoverflow
